examples/experimental/dagster-airlift/examples/tutorial-example/README.md

```diff
@@ -270,76 +270,77 @@ Kicking off a run of the DAG in Airflow, you should see the newly created assets
 
 _Note: There will be some delay between task completion and assets materializing in Dagster, managed by the sensor. This sensor runs every 30 seconds by default (you can reduce down to one second via the `minimum_interval_seconds` argument to `sensor`), so there will be some delay._
 
-## Migrating Assets
+## Proxying to Assets
```

Comment:
So to be clear I still think we should describe this as "Migrating to Assets". Proxying is our _tool_ for facilitating the process.